core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLSnapshotTestRun.java

```diff
@@ -62,30 +64,36 @@ public void testMultiSnapshotOutput() throws Exception {
     String tableName = "SnapshotInputTable";
     ETLStage source = new ETLStage(
       "source",
-      new Plugin("Table", ImmutableMap.<String, String>builder()
-        .put(Properties.Table.NAME, tableName)
-        .put(Properties.Table.PROPERTY_SCHEMA, SCHEMA.toString())
-        .put(Properties.Table.PROPERTY_SCHEMA_ROW_FIELD, "id")
-        .build()));
+      new ETLPlugin("Table", BatchSource.PLUGIN_TYPE,
+                    ImmutableMap.<String, String>builder()
+                      .put(Properties.Table.NAME, tableName)
+                      .put(Properties.Table.PROPERTY_SCHEMA, SCHEMA.toString())
+                      .put(Properties.Table.PROPERTY_SCHEMA_ROW_FIELD, "id")
+                      .build(),
+                    null));
```

Comment:
just a comment for a potential future change - seems like it'd be useful to have a constructor without ArtifactSelector.